### PR TITLE
97668-Final-colors-for-Brand-Alignment

### PR DESCRIPTION
**Pull request by joebri**
97668
### Review
 - [x] @boskya

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -35,7 +35,8 @@
     <div class="daptiv-container">
         {{ content }}
 
-        <footer class="footer">
+
+        <footer class="footer" style="margin: 25px 10px 2px 10px;">
```
Comment (boskya) [inline docs/_layouts/default.html]: you could have inline styles within footer class

File: styles/_colors.scss [hunk 2]
```diff
@@ -2,26 +2,29 @@
...
-$color-black: #333;
-$color-dark-dark-gray: #666;
-$color-dark-gray: #999;
-$color-medium-dark-gray: #b2b2b2;
-$color-medium-gray: #ccc;
-$color-medium-light-gray: #ddd;
-$color-light-medium-gray: #efefef;
-$color-light-gray: #f5f5f5;
-$color-white: #fff;
+// Secondary palette
```
Comment (boskya) [inline styles/_colors.scss]: palette is lot more sophisticated than colors :+1:

**Comment by boskya** [comment]
:shipit:

File: docs/_layouts/default.html (hunk 1 above)
Comment (joebri) [inline docs/_layouts/default.html]: The css looks like it's from a package and I could only find the 'min' version (after an admittedly brief search). I wasn't inclined to update a minified css file. :smile:

Comment (boskya) [inline docs/_layouts/default.html]: https://github.com/daptiv/style-guide/blob/master/docs/docs.scss Just need to add it here.

**Pull request merged**